linetools/spectra/utils.py

```diff
@@ -0,0 +1,66 @@
+""" Methods related to spectra
+"""
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+# Import libraries
+import numpy as np
+import pdb
+
+from astropy import units as u
+
+from .xspectrum1d import XSpectrum1D
+
+def splice_two(spec1, spec2, wvmx=None, scale=1., chk_units=True):
+    """ Combine two overlapping spectra.
+
+    Parameters
+    ----------
+    spec1 : XSpectrum1D
+    spec2 : XSpectrum1D
+      The overlapping spectrum. It should cover wavelengths
+      *longer* than the original spectrum.
```

Comment:
Can you add a check for this in the code below? (i.e. spec1.wv[-1] > spec2.wv[0] and spec1.wv[0] < spec2.wv[-1] )